pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -142,17 +142,28 @@ def cmd(self):
         if self.disable_capture:
             cmd.append("-s")
 
-        if self.processes == -1:
-            cmd.append('-n auto')
-            cmd.append('--dist=loadscope')
-        elif self.processes != 0:
-            cmd.append('-n {}'.format(self.processes))
-            cmd.append('--dist=loadscope')
-
-        if not self.randomize:
-            cmd.append('-p no:randomly')
-        if self.eval_attr:
-            cmd.append("-a '{}'".format(self.eval_attr))
+        if self.xdist_ip_addresses:
+            cmd.append('-d')
+            for ip in self.xdist_ip_addresses.split(' '):
+                cmd.append('--tx ssh=ubuntu@{}//python="source /edx/app/edxapp/edxapp_env; python"//chdir="/edx/app/edxapp/edx-platform"'.format(ip))
+            already_synced_dirs = []
```

Comment:
Not that it matters much in this case, but a set seems more appropriate than a list here